internal/layers/layers.go

```diff
@@ -464,7 +514,7 @@ func containerRootfsPath(vm *uvm.UtilityVM, rootPath string) string {
 	return ospath.Join(vm.OS(), rootPath, guestpath.RootfsPath)
 }
 
-func getScratchVHDPath(layerFolders []string) (string, error) {
+func GetScratchVHDPath(layerFolders []string) (string, error) {
```

Comment:
This is still only called in /internal/layers (unless I'm missing it haha) so does it need to be exported now?